webots_ros2_mavic/CHANGELOG.rst

```diff
@@ -0,0 +1,7 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package webots_ros2_mavic
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1.1.0 (2021-xx-xx)
```

Comment:
Here it seems that the date is missing.
This should be fixed in the master branch first, is it correct?

Reply:
Yes, I started a PR here:
https://github.com/cyberbotics/webots_ros2/pull/251